repose-aggregator/src/docs/asciidoc/_backends/haml/html5/document.html.haml

```diff
@@ -108,6 +108,10 @@
     - unless nofooter
       #footer
         #footer-text
+          - if attr? 'project-version'
+            = precede %(#{attr 'version-label'} #{attr 'project-version'}) do
+              - if (attr? :revnumber) || (attr? 'last-update-label')
+                %br
```

Comment:
How did this get here? Also, why is it showing up in the diff since it's already in master? Gah.